PlannTer/Modules/PlantModule/PlantAddView.swift

```diff
@@ -4,6 +4,7 @@ import SwiftData
 struct PlantAddView: View {
     @Environment(\.modelContext) private var context
     @Bindable var room: RoomModel
```

Comment:
I think that you're not using `@Bindable var room` to make the room, in which user clicked `Add Plant`, the default one displayed in `PlantAddView`. Thus, it may be confusing, because plant will be created in the first room on the list.

Reply:
fixed